app/js/services/individuService.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular.module('ddsCommon').service('IndividuService', function($filter, situationsPro, nationalites) {
-    var statutsSpecifiques = _.filter(situationsPro, 'isStatutSpecifique');
+    var statutsSpecifiques = situationsPro;
```

Comment:
(I agree the whole `situationsPro`/`specificSituations`is weird though, but I don't think this PR is that of a good opportunity to dig into it).